frontend/public/components/storage/shared.ts

```diff
@@ -29,30 +30,33 @@ export const provisionerAccessModeMapping = {
   'pd.csi.storage.gke.io': ['ReadWriteOnce'],
 };
 export const initialAccessModes = ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
-export const accessModeRadios = [
+
+export const accessModeRadios = () => [
```

Comment:
Let's call this `getAccessModeRadios`. It makes it more obvious it's a simple getter function.